### PR TITLE
 Countdown to the New Year: 31 Days of Ember Addons - ember-auto-import (Dec 6th 2019)

### LINKED ISSUE #370: Create a blog post for December 6, 2019 about the `ember-auto-import` addon
## Goal
Write a short blog post for the December blog series that is similar to the [example post]( https://github.com/ember-learn/ember-blog/blob/be8f29ba0ae8b5367d1758b1a150c1b503812a77/source/2019-12-01-countdown-to-the-new-year-ember-sortable.md). You can find more information on how to write your post in our [Contribution guidelines for DecEmber](https://github.com/ember-learn/ember-blog/blob/master/source/december-2019-blog-series-template.md).

## Criteria

-  [ ] file name should 2019-12-DD-countdown-to-the-new-year-ADDON-NAME.md where DD is replaced with the day and ADDON-NAME is replaced with the hyphenated name of the addon
- [ ] The opening sentence should be updated where COUNT is the sequential ordinal word. 

```
<b>This is the COUNT in our DecEmber series- <i>"Countdown to The New Year: 31 Days of 
Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and 
we hope you'll join us for the fun!</b>
```

- [ ] The first h2 should be "Day NUM" where NUM is the same as the DD in the date.
- [ ] the link to the addon should point to the page on Ember Observer
- [ ] There should be two sub sections: (both h3)- "What it does" and "Why I like it" 
- [ ] Include whether or not the addon has been updated yet for Ember Octane- if not, invite contributors to help get the addon Octane ready.
- [ ] The closing sentence can be slightly altered but should feel similar: 
"Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!"

### PR DESCRIPTION
**Pull request by jenweber**
Closes #370

### CODE REVIEW

File: source/2019-12-06-countdown-to-the-new-year-ember-auto-import.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Countdown to The New Year- Ember Auto Import
+author: Jen Weber
+tags: 2019, Addons, DecEmber
+alias: 2019-12-06-countdown-to-the-new-year-ember-auto-import.md
+responsive: true
+---
+
+**This is the sixth in our DecEmber series- <i>"Countdown to The New Year: 31 Days of Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
```
Comment (amyrlam) [inline source/2019-12-06-countdown-to-the-new-year-ember-auto-import.md]:
```suggestion
**This is the sixth in our DecEmber series- "Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
```

File: source/2019-12-06-countdown-to-the-new-year-ember-auto-import.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Countdown to The New Year- Ember Auto Import
```
Comment (amyrlam) [inline source/2019-12-06-countdown-to-the-new-year-ember-auto-import.md]:
```suggestion
title: Countdown to The New Year - Ember Auto Import
```

**Review by amyrlam — APPROVED** [review]
reads great to me 👍

File: source/2019-12-06-countdown-to-the-new-year-ember-auto-import.md (hunk 1 above)
Comment (amyrlam) [reply source/2019-12-06-countdown-to-the-new-year-ember-auto-import.md]: Template: https://github.com/ember-learn/ember-blog/blob/master/source/december-2019-blog-series-template.md

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for this great writeup for December :sparkles:

**Pull request merged**